golem-test-framework/src/components/rdb/docker_postgres.rs

```diff
@@ -40,57 +32,36 @@ impl DockerPostgresRdb {
     const DEFAULT_PASSWORD: &'static str = "postgres";
     const DEFAULT_DATABASE: &'static str = "postgres";
 
-    // TODO: can we simplify this and get rid of local_env (and always use localhost and exposed ports)?
-    pub async fn new(local_env: bool, keep_container: bool, port: Option<u16>) -> Self {
-        let host_port = port.unwrap_or(Self::DEFAULT_PORT);
-        info!("Starting Postgres container, host port {}", host_port);
+    pub async fn new() -> Self {
+        info!("Starting Postgres container");
+
         let database = Self::DEFAULT_DATABASE;
         let password = Self::DEFAULT_PASSWORD;
         let username = Self::DEFAULT_USERNAME;
+        let port = Self::DEFAULT_PORT;
 
-        let name = "golem_postgres";
-        let image = testcontainers_modules::postgres::Postgres::default()
-            .with_tag("12")
```

Comment:
12 is end of lived since November of last year and the caches in blacksmith seem to not include it, making the pulls very slow